Lib/fontbakery/callable.py

```diff
@@ -12,14 +12,8 @@
 Conditions) and MAYBE in *customized* reporters e.g. subclasses.
 
 """
-from __future__ import absolute_import, print_function, unicode_literals
-
 from builtins import object
```

Comment:
All those `from builtins import *` imports were put there by `futurize` to make stuff work on Python 2, those need to go as well, as does `future` itself from the dependencies.

Reply:
Done!